bedrock/mozorg/templates/mozorg/about/webvision/full.html

```diff
@@ -7,13 +7,13 @@
 {% extends "mozorg/about/webvision/summary.html" %}
 
 {% block content %}
-  <main role="main" class="c-webvision-article-full mzp-l-content">
+  <main role="main" class="c-webvision-article-full mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
     <aside class="mzp-l-sidebar">
       <nav class="mzp-c-sidemenu webvision-sidebar">
         <section class="mzp-c-sidemenu-summary mzp-js-toggle" aria-controls="sidebar-menu">
           <h3 class="mzp-c-sidemenu-label">Menu</h3>
           <ul>
-            <li>Mozilla's Webvision</li>
+            <li>Mozilla's Web Vision</li>
```

Comment:
Nit: another spot for a curly apostrophe.
```suggestion
            <li>Mozilla’s Web Vision</li>
```

Reply:
thanks!